digits/config/caffe_option.py

```diff
@@ -126,94 +128,137 @@ def validate_version(cls, executable):
         executable -- path to a caffe executable
         """
         minimum_version = parse_version(0,11,0)
```

Comment:
I thought we agreed we would need different minimum versions for different flavors?